dominion/dominion_ui/package.json

```diff
@@ -40,7 +45,8 @@
     "eslint-plugin-react": "^7.12.4",
     "husky": "^1.3.1",
     "lint-staged": "^8.1.0",
-    "prettier": "^1.16.1"
+    "prettier": "^1.16.1",
+    "webpack-cli": "^3.2.3"
```

Comment:
Are we using any of these dependencies: `react-helmet`, `react-router`, `react-router-dom`, `react-select`, `styled-components` and `webpack-cli`?

Reply:
Missed cleaning these.